tests/test_enable_scipts.py

```diff
@@ -1,25 +1,25 @@
 import json
-import os
+from pathlib import Path
+
+import jinja2
 
-from komodo.data import Data
 from komodo.shell import shell
 
 
-def _create_script(kmd_prefix, kmd_pyver, kmd_release, target, template):
-    data = Data(extra_data_dirs=None)
-    os.mkdir(kmd_release)
-    with open("{}/{}".format(kmd_release, target), "w") as f:
-        f.write(
-            shell(
-                [
-                    "m4 {}".format(data.get("enable.m4")),
-                    "-D komodo_prefix={}".format(kmd_prefix),
-                    "-D komodo_pyver={}".format(kmd_pyver),
-                    "-D komodo_release={}".format(kmd_release),
-                    data.get(template),
-                ]
-            ).decode("utf-8")
-        )
+def _create_script(kmd_prefix, _kmd_pyver, kmd_release, target, template):

```

Comment:
Wondering if it would be 'safer' to have a function in `cli.py` (or elsewhere) that `cli.py` and this test both call? Because it seems like otherwise the test and the actual code could diverge?

Reply:
Yes, agreed!